### PR TITLE
tuir: init at 1.28.03

### PR DESCRIPTION
**Pull request by Br1ght0ne**
###### Motivation for this change
`rtv` is [no longer maintained](https://github.com/michael-lazar/rtv/issues/696). `tuir` is one of the [suggested alternatives](https://github.com/michael-lazar/rtv/blob/master/ALTERNATIVES.md#alternative-projects), seems to be maintained and advancing.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @matthiasbeyer @wedens 

Feel free to let me know if you want to also maintain this package.

---
This change is [Reviewable](https://reviewable.io/reviews/nixos/nixpkgs/75817)

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
Sad to see rtv die. Is there a process to deprecate the package in nixpkgs?

Feel free to add me as maintainer!

**Review by matthiasbeyer — APPROVED** [review]

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]
Please rewrite the commit with my correct email address (see my profile).

**Comment by wedens** [comment]
@filalex77 Just curious, is there any particular reason you decided to go with tuir and not [https://github.com/tildeclub/ttrv] which seems to be another active rtv fork?

(I didin't know about both of them until now)

File: pkgs/applications/misc/tuir/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+with python3Packages;
+buildPythonApplication rec {
+  pname = "tuir";
+  version = "1.28.03";
+
+  src = fetchFromGitLab {
+    owner = "ajak";
+    repo = pname;
+    rev = "16f45bacfb845fffdc7a0d3941c83ec6a06ae7e2";
```
Comment (sikmir) [inline pkgs/applications/misc/tuir/default.nix]: Why not `rev = "v${version}";`?

File: pkgs/applications/misc/tuir/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{ lib, fetchFromGitLab, python3Packages }:
+
+with python3Packages;
+buildPythonApplication rec {
+  pname = "tuir";
+  version = "1.28.03";
```
Comment (sikmir) [inline pkgs/applications/misc/tuir/default.nix]: Fix version to `1.28.3`.

File: pkgs/applications/misc/tuir/default.nix [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  # Tests try to access network
+  doCheck = false;
+
+  checkPhase = ''
+    py.test
+  '';
+
+  checkInputs = [ coverage coveralls docopt mock pylint pytest vcrpy ];
+
+  propagatedBuildInputs = [ beautifulsoup4 decorator kitchen requests six ];
```
Comment (sikmir) [inline pkgs/applications/misc/tuir/default.nix]: `six` is not required here, as far as Python 3 is used.

**Review by sikmir — CHANGES_REQUESTED** [review]

**Comment by filalex77** [comment]
Thanks @sikmir!

**Review by sikmir — APPROVED** [review]

**Pull request merged**